### DISCUSSION TITLE
大佬们，求教一个

### DISCUSSION THREAD

**Discussion by qiangua5210** [vernesong/OpenClash#3582] [General]
我用的vmess节点，之前在openclash里面是正常使用的， 然后突然就不行，但是在ShadowSocksR Plus+或者手机上都是正常使用的，ip和域名都没有被墙，之前也出现这个问题， 我更换了固件也不行，用的tun内核，然后不行了之后换成meta内核一样不行

**Comment by vernesong** [comment]
跳过验证，服务器地址尝试填ip
